src/frontend/utils/errors/exception.ts

```diff
@@ -0,0 +1,6 @@
+// To use for default statement in switch where all cases must be covered
```

Comment:
in the commit message (frontend) add an exception to type check enum with (no s)

switch case**.**